docs/en/connector-v2/source/Mysql.md

```diff
@@ -114,9 +114,13 @@ sink {
 > Read your query table in parallel with the shard field you configured and the shard data  You can do this if you want to read the whole table
 
 ```
+env {
+  execution.parallelism = 10
```

Comment:
Modifying this requires specifying `partition_column` otherwise it has no effect.

Reply:
Done

Reply:
![image](https://github.com/apache/seatunnel/assets/76689593/304bf9f6-2477-4ed0-b8f8-d7f397a83c85)
Because it's hidden you can't see it when you unfold it

Reply:
![Uploading image.png…]()